corpus/expressions.txt

```diff
@@ -1001,8 +1001,8 @@ b = <Text {...j}></Text>
     (identifier)
     (jsx_element
       (jsx_opening_element (identifier)
-        (jsx_attribute (property_identifier))
-        (jsx_attribute (property_identifier) (jsx_expression (identifier))))
+        (jsx_attribute (identifier))
+        (jsx_attribute (identifier) (jsx_expression (identifier))))
```

Comment:
Ideally, I'd like for these property names to still be called `property_identifier`. It's just a naming thing, but currently, the grammar has a nice property that the identifiers are neatly categorized with regard to which ones are variable reference (`identifier`), which are names of object properties (`property_identifier`), which are type names `type_identifier`.